### ISSUE TITLE
nativeGenerateVideoWithFilter applying multiple filter

### ISSUE THREAD

**Issue by ksc91u** [wysaid/android-gpuimage-plus#342]
I wish to apply blendImage to my video, and apply filterA for 10 frames, filterB for 10 frames, filterC for 10 frames and so on.

Is it possible with this library?

Thanks 大師兄

**Comment by ksc91u** [comment]
Looks like I have to change cgeGenerateVideoWithFilter to make this work.

**Comment by wysaid** [comment]
No, you can write a fake filter to wrap your logic like the [CGEMutipleEffectFilter](https://github.com/wysaid/android-gpuimage-plus/blob/master/library/src/main/jni/include/filters/cgeMultipleEffects.h#L43)
